builtin/providers/datadog/resource_datadog_timeboard_test.go

```diff
@@ -62,6 +62,56 @@ resource "datadog_timeboard" "acceptance_test" {
 }
 `
 
+const config3 = `
+resource "datadog_timeboard" "acceptance_test" {
+  title = "Acceptance Test Timeboard"
+  description = "Created using the Datadog prodivider in Terraform"
```

Comment:
That's what I get for copy-pasting from the existing tests 👎 but I fixed it now.